src/pybind/mgr/dashboard/controllers/_crud.py

```diff
@@ -90,33 +90,38 @@ class FormField(NamedTuple):
     default_value: Optional[Any] = None
     optional: bool = False
     html_class: str = ''
-    label_html_class: str = 'col-form-label'
-    field_html_class: str = 'col-form-input'
 
     def get_type(self):
         _type = ''
         if self.field_type == str:
-            _type = 'string'
+            _type = 'input'
         elif self.field_type == int:
-            _type = 'integer'
+            _type = 'input'
         elif self.field_type == bool:
-            _type = 'boolean'
+            _type = 'input'
+        elif self.is_enum():
+            _type = 'input'
         else:
             raise NotImplementedError(f'Unimplemented type {self.field_type}')
         return _type
 
+    def is_enum(self):
+        return self.field_type == Enum or \
+            (isclass(self.field_type) and issubclass(self.field_type, Enum))
```

Comment:
I have to remove this, I was working on adding it to rgw roles and this spilled over to this pr